test/e2e/cloud/gcp/kubelet_security.go

```diff
@@ -48,8 +48,8 @@ var _ = SIGDescribe("Ports Security Check [Feature:KubeletSecurity]", func() {
 	})
 
 	// make sure kubelet readonly (10255) and cadvisor (4194) ports are disabled via API server proxy
-	ginkgo.It(fmt.Sprintf("should not be able to proxy to the readonly kubelet port %v using proxy subresource", ports.KubeletReadOnlyPort), func(ctx context.Context) {
-		result, err := e2ekubelet.ProxyRequest(f.ClientSet, nodeName, "pods/", ports.KubeletReadOnlyPort)
+	ginkgo.It(fmt.Sprintf("should not be able to proxy to the readonly kubelet port 10255 using proxy subresource"), func() {
```

Comment:
fmt.Sprintf is not needed since you removed the substitute

Reply:
updated.